tests/README.md

```diff
@@ -80,16 +80,15 @@ In addition to standard go tests parameters, the following custom parameters are
  Parameter | Description | Possible values | Default
  --- |--- | --- | ---
 rook_platform| platform rook needs to be installed on  | kubernetes | kubernetes
-k8s_version  | version of Kubernetes to be installed  | v1.5+  | v1.7.5
+k8s_version  | version of Kubernetes to be installed  | v1.6+  | v1.8.5
 rook_image | rook image name to be installed | valid image name | rook/rook
 toolbox_image | toolbox image name to be installed | valid image name | rook/toolbox
 skip_install_rook | skips installing rook (if already installed) | true or false  | false
-load_parallel_runs | performs concurrent operations (optional used for load test) | any number | 20
-
-If the `install_rook` flag is set to false, then all the other flags are ignored
-and tests are run without rook being installed and setup. Use this flag to run tests against
-a pre-installed/configured rook.
-
+load_parallel_runs | performs concurrent operations (optional used for load test only) | any number | 20
+load_volumes | number of volumes ( block,object store or shared file systems) to be created during load test (optional used for load test only)| >1 | 1
+load_time | number of seconds load test run for (optional used for load test only) | >1 | 1800
+load_size | load profile - choice S,M or L(3M,10M and 50M per thread - optional used for load test only) | S,M or L | M
```

Comment:
what about this description? 
`load_size | size of profile to load (3M, 10M, or 50M per thread - load test only) | S, M, or L | M`

Reply:
where do those sizes come from? Should you just allow the number to be used directly instead of using S,M,L?